Kryptonite/KRBaseController.swift

```diff
@@ -65,9 +70,30 @@ class KRBaseController: UIViewController {
     func shouldPostAnalytics() -> Bool {
         return true
     }
+    
+    func approveControllerDismissed(allowed:Bool) {
+        let result = allowed ? "allowed" : "rejected"
```

Comment:
can we refactor this out into a single method and call it from each class that uses it?